deploy/template.yaml

```diff
@@ -148,6 +148,18 @@ Resources:
             Condition:
               StringEquals:
                 "aws:SourceAccount": !Sub "${AWS::AccountId}"
+          - Effect: Allow
```

Comment:
Doesn't this need to be a Deny given the way the conditions are written or have I misunderstood this statement? For example Deny if securetTransport is false and if TLS is less than 1.2??